lib/types/any/index.js

```diff
@@ -106,8 +107,8 @@ module.exports = internals.Any = class {
         obj._type = this._type;
         obj._settings = internals.concatSettings(this._settings);
         obj._baseType = this._baseType;
-        obj._valids = Hoek.clone(this._valids);
-        obj._invalids = Hoek.clone(this._invalids);
+        obj._valids = this._valids;
+        obj._invalids = this._invalids;
```

Comment:
Yes, Hoek.clone is relatively expensive, but it's there to prevent modifications of the schema internals that could come from passing by reference.

Reply:
The `clone()` is actually a bug. See https://github.com/hapijs/joi/pull/1414.